[CI] LifecycleLicenseIT. testSearchableSnapshotActionErrorsOnInvalidLicense() failure

**Build scan**: https://gradle-enterprise.elastic.co/s/6shbgmna4stpi/console-log?task=:x-pack:plugin:ilm:qa:multi-node:javaRestTest

**Repro line**: `./gradlew ':x-pack:plugin:ilm:qa:multi-node:javaRestTest' --tests "org.elasticsearch.xpack.ilm.LifecycleLicenseIT.testSearchableSnapshotActionErrorsOnInvalidLicense" -Dtests.seed=EC351B0554B89297 -Dtests.security.manager=true -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=sl -Dtests.timezone=America/Indiana/Winamac -Druntime.java=11 -Dlicense.key="x-pack/license-tools/src/test/resources/public.key"`

**Reproduces locally?**: Yes!

**Applicable branches**: master, 7.x

(looks like release builds only)

**Failure history**: (for more than 7 days)

https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=Europe/Amsterdam&tests.container=org.elasticsearch.xpack.ilm.LifecycleLicenseIT&tests.sortField=FAILED&tests.test=testSearchableSnapshotActionErrorsOnInvalidLicense&tests.unstableOnly=true

**Failure excerpt**:

```
org.elasticsearch.xpack.ilm.LifecycleLicenseIT > testCreatePolicyUsingActionAndNonCompliantLicense FAILED |  
-- | --
  | java.lang.AssertionError: |  
  | Expected: a string containing "\"type\" : \"trial\"" ignoring case |  
  | but: was "{ |  
  | "license" : { |  
  | "status" : "active", |  
  | "uid" : "166fb2b1-d9e4-4c73-a373-8139ffae327e", |  
  | "type" : "basic", |  
  | "issue_date" : "2020-12-09T09:31:42.684Z", |  
  | "issue_date_in_millis" : 1607506302684, |  
  | "max_nodes" : 1000, |  
  | "max_resource_units" : null, |  
  | "issued_to" : "javaRestTest", |  
  | "issuer" : "elasticsearch", |  
  | "start_date_in_millis" : -1 |  
  | } |  
  | } |  
  | " |  
  | at __randomizedtesting.SeedInfo.seed([EC351B0554B89297:AEAE737A0BA2545B]:0) |  
  | at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18) |  
  | at org.junit.Assert.assertThat(Assert.java:956) |  
  | at org.junit.Assert.assertThat(Assert.java:923) |  
  | at org.elasticsearch.xpack.ilm.LifecycleLicenseIT.lambda$checkCurrentLicenseIs$5(LifecycleLicenseIT.java:147) |  
  | at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:953) |  
  | at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:926) |  
  | at org.elasticsearch.xpack.ilm.LifecycleLicenseIT.checkCurrentLicenseIs(LifecycleLicenseIT.java:143) |  
  | at org.elasticsearch.xpack.ilm.LifecycleLicenseIT.resetLicenseToTrial(LifecycleLicenseIT.java:61)


```

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/LifecycleLicenseIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/LifecycleLicenseIT.java
@@ -7,7 +7,6 @@
 package org.elasticsearch.xpack.ilm;
 
 import org.apache.http.util.EntityUtils;
-import org.apache.lucene.util.LuceneTestCase;
 import org.elasticsearch.client.Request;
 import org.elasticsearch.client.Response;
 import org.elasticsearch.client.ResponseException;
@@ -44,7 +43,6 @@ import static org.hamcrest.CoreMatchers.containsStringIgnoringCase;
 import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 import static org.hamcrest.Matchers.is;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/66088")
 public class LifecycleLicenseIT extends ESRestTestCase {
 
     private String policy;

